target/linux/ramips/dts/mt7621_asus_4g-ax56.dts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "asus,4g-ax56", "mediatek,mt7621-soc";
+	model = "ASUS 4G-AX56";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+		bootargs-override = "console=ttyS0,115200";
```

Comment:
what happen here? why both override and bootargs?

Reply:
this was likely just copied over from other device trees
https://github.com/Znevna/openwrt/commit/8c00fd9b4519bf0ef8fb3470a6df421b9f38c03c